src/components/layers-panel/arcgis-import-panel/arcgis-import-panel.tsx

```diff
@@ -0,0 +1,302 @@
+  const loadingStatus = useAppSelector(selectStatus);
+
+  const isLoading = loadingStatus === "loading";
+
+  useEffect(() => {
+    dispatch(resetArcGisContentSelected());
+  }, []);
+
+  const handleImport = () => {
+    const arcGisItem = arcGisContentArray.find(
+      (item) => item.id === arcGisContentSelected
+    );
+    if (arcGisItem) {
+      onImport(arcGisItem);
+    }
+  };
+
+  const onSort = (dataColumnName: ArcGisContentColumnName) => {
+    if (sortColumn === dataColumnName) {
+      dispatch(setSortAscending(!sortAscending));
+    } else {
+      dispatch(setSortColumn(dataColumnName));
+    }
+  };
+
+  const formatDate = (date: number) => {
+    const formatter = new Intl.DateTimeFormat("en-US", {
+      month: "long",
+      day: "2-digit",
+      year: "numeric",
+    });
+    return formatter.format(date);
+  };
+
+  const theme = useTheme();
+  return (
+    <ModalDialog
+      title={"Select map to import"}
+      okButtonText={"Import Selected"}
+      onConfirm={handleImport}
+      onCancel={onCancel}
+    >
+      <Table>
+        <TableHeader>
+          <TableHeaderItem1></TableHeaderItem1>
+          <TableHeaderItem2>
+            <TitleCellContainer onClick={() => onSort("title")}>
+              Title
+              <IconContainer enabled={sortColumn === "title"}>
+                <ExpandIcon
+                  expandState={ExpandState.expanded}
+                  collapseDirection={
+                    sortAscending
+                      ? CollapseDirection.top
+                      : CollapseDirection.bottom
+                  }
+                  onClick={() => onSort("title")}
+                  fillExpanded={theme.colors.buttonDimIconColor}
+                  width={6}
+                />
+              </IconContainer>
+            </TitleCellContainer>
+          </TableHeaderItem2>
+
+          <TableHeaderItem3>
+            <TitleCellContainer onClick={() => onSort("created")}>
+              Date
+              <IconContainer enabled={sortColumn === "created"}>
+                <ExpandIcon
+                  expandState={ExpandState.expanded}
+                  collapseDirection={
+                    sortAscending
+                      ? CollapseDirection.top
+                      : CollapseDirection.bottom
+                  }
+                  onClick={() => onSort("created")}
+                  fillExpanded={theme.colors.buttonDimIconColor}
+                  width={6}
+                />
+              </IconContainer>
+            </TitleCellContainer>
+          </TableHeaderItem3>
+        </TableHeader>
+
+        <TableContent key={`${sortColumn}-${sortAscending}`}>
+          <SpinnerContainer visible={isLoading}>
+            <LoadingSpinner />
+          </SpinnerContainer>
+
+          {arcGisContentArray.map((contentItem) => {
+            const isMapSelected = arcGisContentSelected === contentItem.id;
+
+            return (
+              <Fragment key={contentItem.id}>
```

Comment:
TableRow is a container. You don't need `Fragment`

Reply:
Removed.